test/unit/models/forms/v2/LiabilityDetailsFormSpec.scala

```diff
@@ -164,7 +168,7 @@ class LiabilityDetailsFormSpec extends ModelsBaseSpec {
         val form = liabilityDetailsForm.liabilityDetailsForm(sampleCase)
 
         form.hasErrors shouldBe false
-        form.data      shouldBe params.mapValues(v => v.head)
+        form.data      shouldBe params.map { case (fst, snd) => fst -> snd.head }
```

Comment:
Same with the others as it can just be `params.view.mapValues(v => v.head).toMap`. Thanks